src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/EquippableModule.java

```diff
@@ -0,0 +1,53 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import ch.njol.skript.registrations.Classes;
+import ch.njol.skript.util.ItemSource;
+import ch.njol.skript.util.slot.Slot;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.components.EquippableComponent;
+import org.skriptlang.skript.addon.AddonModule;
+import org.skriptlang.skript.addon.SkriptAddon;
+import org.skriptlang.skript.lang.converter.Converter;
+import org.skriptlang.skript.lang.converter.Converters;
+
+import java.io.IOException;
+
+public class EquippableModule implements AddonModule {
+
+	@Override
+	public boolean canLoad(SkriptAddon addon) {
+		return Skript.classExists("org.bukkit.inventory.meta.components.EquippableComponent");
+	}
+
+	@Override
+	public void init(SkriptAddon addon) {
+		Classes.registerClass(new ClassInfo<>(EquippableWrapper.class, "equippablecomponent")
+			.user("equippable ?components?")
+			.name("Equippable Components")
+			.description("Represents an equippable component used for items.")
+			.requiredPlugins("Minecraft 1.21.2+")
+			.since("INSERT VERSION")
+			.defaultExpression(new EventValueExpression<>(EquippableWrapper.class))
+			.cloner(EquippableWrapper::clone)
```

Comment:
i've mentioned this a bit in discussion around exprcopy but adding this cloner means every time you set a component to a var, it will be cloned, which makes the copy expression rather redundant.